_translator-files/po/de/Installation-for-Windows.po

```diff
@@ -42,7 +43,7 @@ msgstr "Installation auf Windows"
 #. type: Plain text
 #: ../wiki/en/Installation-for-Windows.md:9
 msgid "{% include breadcrumb.html root=\"Using Jamulus\" branch1=\"Getting Started\" branch1-url=\"Getting-Started\" %}"
-msgstr "{% include breadcrumb.html root=\"Jamulus Nutzen\" branch1=\"Erste Schritte\" branch1-url=\"Getting-Started\" %}"
+msgstr "{% include breadcrumb.html root=\"Using Jamulus\" branch1=\"Getting Started\" branch1-url=\"Getting-Started\" %}"
```

Comment:
@gilgongo what needs to be written here? Isn't it translatable, so Getting Started needs to be translated too (Erste Schritte and Jamulus Nutzen)